.github/workflows/scripts/prs/detectTargetBranch.js

```diff
@@ -22,90 +22,46 @@ module.exports = async ({ core, context, github }) => {
+      core.setOutput('TARGET_BRANCHES', '');
       return;
     }
 
     core.info(`>>> Target labels found: ${targetLabels.join(', ')}`);
-    let target = '';
-
-    // there was a target branch label present
-    // filter the highest available target number and remove the others from the PR when present
-    if (targetLabels.length > 1) {
-      core.info(`>>> Multiple target labels found.`);
-      targetLabels.sort((a, b) => {
-        const aNum = parseInt(a.match(/\d+/)[0], 10);
-        const bNum = parseInt(b.match(/\d+/)[0], 10);
-        return bNum - aNum;
-      });
-
-      target = targetLabels.shift();
-
-      core.info(`>>> Sorting and setting the highest as 'TARGET_BRANCH' output.`);
-      core.setOutput('TARGET_BRANCH', target);
-
-      // since we have multiple targets we need to add the "needs cherry-pick" label
-      // this makes this workflow de-facto recursive
-      transferLabels.push('needs cherry-pick');
 
-      // add the other targets to the transfer labels
-      transferLabels.push(...targetLabels);
-      core.setOutput('TRANSFER_LABELS', transferLabels.join(','));
-
-      // the others will be removed from the PR
-      core.info(`>>> Removing the other target labels from the PR`);
-      for (const label of targetLabels) {
-        await github.rest.issues.removeLabel({
-          owner,
-          repo,
-          issue_number: pullNumber,
-          name: label,
-        });
-      }
-
-      core.info(`>>> Creating explanatory comment on PR`);
-      await github.rest.issues.createComment({
-        owner,
-        repo,
-        issue_number: pullNumber,
-        body: [
-          `The target branch for the cherry-pick PR has been set to \`${target}\`.`,
-          `Branches that will be created after merging are: ${targetLabels.join(', ')}`,
-          `Thank you!`,
-        ].join('\n\n'),
-      });
-      return;
-    }
+    // get a list of the original reviewers
+    const reviewers = pr.requested_reviewers.map((reviewer) => reviewer.login);
+    core.info(`>>> Reviewers from original PR: ${reviewers.join(', ')}`);
 
-    core.info(`>>> Removing the "needs cherry-pick" label from the PR`);
-    await github.rest.issues.removeLabel({
+    core.info(`>>> Creating explanatory comment on PR`);
+    await github.rest.issues.createComment({
       owner,
       repo,
       issue_number: pullNumber,
-      name: 'needs cherry-pick',
+      body: `Cherry-pick PRs will be created targeting branches: ${targetLabels.join(', ')}`,
     });
 
-    target = targetLabels[0];
-    core.info(`>>> Setting found 'TARGET_BRANCH' output.`);
-    core.setOutput('TARGET_BRANCH', target);
-    core.setOutput('TRANSFER_LABELS', transferLabels.join(','));
+    // set the target branches as output to be used as an input for the next step
+    core.setOutput('TARGET_BRANCHES', targetLabels.join(','));
+    core.setOutput('LABELS', ['cherry-pick', ...otherLabels].join(','));
```

Comment:
Should we remove the `needs cherry-pick`?

Reply:
mmh, good question ... maybe not, since otherwise we have no indication if it got cherry picked or not

Reply:
I'll probably add creating messages with the new cherry pick PRs to the OG PR in a next step, but for now lets keep it

Reply:
My point is more that the new PR will have the `needs cherry-pick` label, which could trigger this workflow again

Reply:
done here: https://github.com/mui/mui-public/pull/234/commits/7df7fc425dbe8a15492df102959e5283abd3a6c2